### ISSUE TITLE
Model Generation?

### ISSUE THREAD

**Issue by deepWhomst** [deeppomf/DeepCreamPy#20]
Any chance of getting the model generation code available here or in a separate repo? I've made my own model and driver python that parses for 0,0,255 as nipples to allow chaining uncensoring models, but I'm having trouble getting my model to work correctly with Keras. Would it be possible to have access to your training notebook? I'm not asking for the database you used to create the model, just the model training notebook. Thank you for making this!

**Comment by deeppomf** [comment]
See https://github.com/MathiasGruber/PConv-Keras

**Comment by deepWhomst** [comment]
Thanks! I'll check it out. The training script looks a lot cleaner than what I was doing for sure.

**Comment by deepWhomst** [comment]
Go ahead and close.

**Comment by deepWhomst** [comment]
If I do get the nipple decensor working, would you be interested in merging a PR that adds chained mask processing of the images? I'm trying to decide how clean my code needs to be.

**Comment by deeppomf** [comment]
That'd be great. But I will need to spend a few days rethinking the license.
I do want to sell this project (will still be available free here on github), but the current license is too hostile to contributions.
